cypress/support/server/Networkshim.js

```diff
@@ -0,0 +1,151 @@
+import { getFileName } from './utils.js'
```

Comment:
Remove all code related to the network shim, since it isn't in use and may never be.

Reply:
does that mean removing the whole `server/enableNetworkShim.js` file as well then?